packages/account/src/Sections/Security/Passkeys/passkeys.tsx

```diff
@@ -37,18 +38,30 @@ const Passkeys = observer(() => {
     });
 
     React.useEffect(() => {
-        if (!passkeys_list?.length && !is_passkey_registered) {
+        if (is_passkeys_list_loading) return;
+        if (!passkeys_list?.length) {
```

Comment:
cant we add this as an else statement to first if block?

Reply:
yes, @amina-deriv , we can do it. I just try to be more explicit here: if there is loading - do not run useEffect logic.
and if we add 'else', we also need 'if'. so it will just increase the quantity of lines